creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -116,6 +119,21 @@ const handleSyncFromPrimary = async ({
         }
       }
 
+      try {
+        const numFilesDeleted = await deleteAllCNodeUserDataFromDisk(
+          wallet,
+          logger
+        )
+        logger.info(`Deleted ${numFilesDeleted} for ${wallet}`)
+      } catch (error) {
+        errorResponse = {
+          error,
+          result: 'failure_delete_disk_data'
+        }
+
+        throw error
+      }
+
```

Comment:
main thing here is doing this after DB delete as we'd discussed

Reply:
ah right, glad you caught that early - fixing